SafeFoodie_Fresh/lib/screens/account.dart

```diff
@@ -133,6 +121,41 @@ class _AccountState extends State<Account> {
                 leading: Icon(Icons.phone, color: Colors.green),
                 onPressed: (BuildContext context) {},
               ),
+               SettingsTile(
+                title: const Text('Logout', style: TextStyle(color: Colors.green)),
+                value: const Text('Logout of account',
+                    style: TextStyle(color: Color.fromARGB(216, 230, 182, 53))),
+                leading: const Icon(Icons.waving_hand_outlined, color: Colors.green),
+                trailing: const Icon(Icons.logout_sharp ,
+                    color: Color.fromARGB(216, 230, 182, 53)),
+                onPressed: (BuildContext context) {
+                  showDialog(
+                      context: context,
+                      builder: (BuildContext context) {
+                        return AlertDialog(
+        title: Text('Are you sure you want to logout?'),
+        actions: <Widget>[
+          TextButton(
+            child: Text('Cancel'),
+            onPressed: () {
+              Navigator.of(context).pop();
+            },
+          ),
+          TextButton(
+            child: Text('Confirm'),
+            onPressed: () async {
+               Navigator.popUntil(
```

Comment:
The original code was working on this, with these two added lines the logout bug should be resolved.

Reply:
Awesome! we were stumped on this for a while.